### PR TITLE
Add badge for build and link to S3 to README.md

### PR DESCRIPTION
**Pull request by jenow**
## What does this PR do ?

Added the build badge and the links to download tgz archive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,18 +17,27 @@ The complete SDK documentation is available [here](http://docs.kuzzle.io/sdk-ref
 
 The C SDK implement the websocket protocol.
```
Comment (scottinet) [inline README.md]: implements

File: README.md [hunk 2]
```diff
@@ -15,18 +17,27 @@ The complete SDK documentation is available [here](http://docs.kuzzle.io/sdk-ref
...
 ### Clone the depot
 
 ```sh
 git clone --recursive git@github.com:kuzzleio/sdk-c.git
-```
-
-### Build
-
-```
+cd sdk-go
```
Comment (scottinet) [inline README.md]: cd sdk-c

File: README.md [hunk 3]
```diff
@@ -15,18 +17,27 @@ The complete SDK documentation is available [here](http://docs.kuzzle.io/sdk-ref
 
 The C SDK implement the websocket protocol.
 
-## Installation
+## Build
 
 ### Clone the depot
```
Comment (scottinet) [inline README.md]: See my comment in the Java SDK PR

File: README.md [hunk 4]
```diff
@@ -15,18 +17,27 @@ The complete SDK documentation is available [here](http://docs.kuzzle.io/sdk-ref
...
-
-### Build
-
-```
+cd sdk-go
+git submodule update --init --recursive
 make
 ```
 
 You should know have the SDK in the build directory.
```
Comment (scottinet) [inline README.md]: You should now

File: README.md [hunk 5]
```diff
@@ -15,18 +17,27 @@ The complete SDK documentation is available [here](http://docs.kuzzle.io/sdk-ref
...
+
+You can find prebuilt SDK's for each version and architecture:
+
+arm64: https://dl.kuzzle.io/sdk/c/master/kuzzlesdk-cpp-aarch64-1.0.0.tar.gz
+
+arm32: https://dl.kuzzle.io/sdk/c/master/kuzzlesdk-cpp-armhf-1.0.0.tar.gz
+
+amd64: https://dl.kuzzle.io/sdk/c/master/kuzzlesdk-cpp-amd64-1.0.0.tar.gz
+
+i386:  https://dl.kuzzle.io/sdk/c/master/kuzzlesdk-cpp-i386-1.0.0.tar.gz
```
Comment (scottinet) [inline README.md]: i386 => x86

**Review by scottinet — CHANGES_REQUESTED** [review]

**Review by Aschen — APPROVED** [review]

**Review by benoitvidis — COMMENTED** [review]
I am a little concerned that we're going to miss the dl urls updates.
Shouldn't we wait to have a `xxx/latest/xxx` url before including the links?

**Comment by jenow** [comment]
@benoitvidis I agree but this does not exist on our s3

**Review by scottinet — APPROVED** [review]

**Pull request merged**